### ISSUE TITLE
Please Update to Latest Applovin Max Sdk 11.4.4

### ISSUE THREAD

**Issue by karanchaurasiya** [DrMoriarty/godot-applovin-max#21]
Sir it has old sdk please update to latest one applovin max sdk 11.4.4

**Comment by basdp** [comment]
@DrMoriarty I see it has been updated to recent version on Android, but not on iOS. Do you accept pull requests for new features, updates and other enhancements like newer versions?

**Comment by finepointcgi** [comment]
Yes he does I have contributed before I dont have access to IOS or i would have updated it @basdp
